pygments/lexers/c_like.py

```diff
@@ -657,3 +657,82 @@ class OmgIdlLexer(CLexer):
             include('annotation_appl'),
         ],
     }
+
+
+class PromelaLexer(CLexer):
+    """
+    For the Promela language used with SPIN.
+    """
+    
+    name = 'Promela'
+    aliases = ['promela']
+    filenames = ['*.pml', '*.prom', '*.prm', '*.promela', '*pr', '*.pm']
```

Comment:
Shouldn't this be

```suggestion
    filenames = ['*.pml', '*.prom', '*.prm', '*.promela', '*.pr', '*.pm']
```

?

Reply:
Good catch. That was a typo.